.github/workflows/tests.yaml

```diff
@@ -28,11 +28,11 @@ jobs:
       run: docker build -t civiform-dev --cache-from docker.io/civiform/civiform-dev:latest ./
     - name: Run tests
       run: USE_LOCAL_CIVIFORM=true bin/run-test-ci
-    - name: jacoco-badge-generator
-      uses: cicirello/jacoco-badge-generator@v2
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
```

Comment:
Why was it bad to save the generated badge to the repository?  It would reduce dependance on third-party tools

Reply:
Our repo has a lot of protections (like Reviewers approval required) hence this approach did not work.